lib/cfndsl.rb

```diff
@@ -88,13 +88,12 @@ def self.eval_file_with_extras(filename, extras = [], logstream = nil)
   end
 end
 
-def CloudFormation(&block)
-  x = CfnDsl::CloudFormationTemplate.new
-  x.declare(&block)
```

Comment:
Don't have the right laptop to look at rest of the code. Is this the only place we call declare? If not they'll need to be removed as well